src/V3Randomize.cpp

```diff
@@ -718,6 +718,37 @@ class ConstraintExprVisitor final : public VNVisitor {
             initTaskp->addStmtsp(methodp->makeStmt());
         }
     }
+    void visit(AstCountOnes* nodep) override {
+        // Convert it to (x & 1) + ((x & 2) >> 1) + ((x & 4) >> 2) + ...
+        FileLine* const fl = nodep->fileline();
+        AstNodeExpr* const argp = nodep->lhsp()->unlinkFrBack();
+        AstNodeExpr* sump
+            = new AstAnd{fl, argp, new AstConst{fl, AstConst::WidthedValue{}, argp->width(), 1}};
+        sump->user1(true);
+        for (int i = 1; i < argp->width(); i++) {
+            AstAnd* const andp = new AstAnd{
+                fl, argp->cloneTreePure(false),
+                new AstConst{fl, AstConst::WidthedValue{}, argp->width(), (uint32_t)1 << i}};
```

Comment:
```suggestion
                new AstConst{fl, AstConst::WidthedValue{}, argp->width(), (uint32_t)1 << i}};
```
This won't work for more than 32 bit numbers, form a V3Number than pass that.  Similar below.  Please test an e.g. 96 bit $countones, or if for some reason the solver will choke for other reasons need a user error message and comment.

Reply:
```suggestion
               V3Number num{nodep, argp->width(), 0};
               num.setBit(i, 1);
               new AstConst{fl, num};
```

Reply:
I created such a test and it passes, but it takes 1m14s to complete. I committed a test with 33 bit variable instead. It fails without these changes too, but takes much less time.